cartridges/int_cloudinary/cartridge/templates/default/include/cloudinaryStaticLibs.isml

```diff
@@ -2,8 +2,8 @@
 <isif condition="${preferences.CLDEnabled}">
     <isset name="cloudinaryConstants" value="${require('*/cartridge/scripts/util/cloudinaryConstants')}" scope="page" />
     <isset name="CLDImagePageTypeSettings" value="${JSON.parse(preferences.CLDImagePageTypeSettings)}" scope="page"/>
-    <isif condition="${cloudinaryConstants.CLD_DOMAIN}">
-        <link rel="preconnect" href="${'https://' + cloudinaryConstants.CLD_DOMAIN}" />
+    <isif condition="${cloudinaryConstants.CLD_DOMAIN && (pdict.cloudinary.galleryEnabled || pdict.cloudinary.isGalleryEnabled)}">
```

Comment:
@asad-rafter this condition already exists lower in this code, please add the line there